app/constants/urls.js

```diff
@@ -20,6 +20,10 @@ export const AUTH = {
   SIGN_UP: 'https://my.realdevsquad.com/signup',
 };
 
+export const API_URL = {
+  JOIN_URL: 'https://api.realdevsquad.com/users/self/intro',
+};
```

Comment:
Make a new constants file, for API URLs named as `APIs.js` and import the BASE_URL from the `environment.js` then use it something like this
```JS
import ENV from 'config/environments.js`

BASE_API_URL = ENV.BASE_API_URL
const JOIN_URL = `${BASE_API_URL}/users/self/intro`
```

The code is not exactly same as in the working state, it's there for reference only